DTS-STN/Service-Canada-Labs
# Signup Review page

# Description

Created signup review page where a user can review their entered signup information before submitting registration. Users can go back to the form and it will be populated with their previously entered data which can be modified before the user returns to submitting their information.

Form data is held in sessionstorage until the user submits registration or exits the tab, page, browser or clears the form at which points data will be removed.

Labels in the left column of the review page tables are using the SignupPage content returned from graphQL, as are the value labels.

Please give it a go and let me know of any issues you find, this one has a couple moving parts.

## Acceptance Criteria

User should be able to review their data before submission. If a user wants to modify data entered, they should be able to return to the signup page and do that without reentering all information. 

**Missing french translations for content**

**Missing content for metadata fields**

[Figma file](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?node-id=7953%3A21831)

![Screen Shot 2022-09-09 at 12 54 17 PM](https://user-images.githubusercontent.com/31868510/189432939-2417e497-f6d6-4ef3-ac4c-987c764d74ca.png)

## Test Instructions

1. Go to signup page and enter values in different configurations
2. Submit the signup form
3. See review page and see the information you entered
4. Either click "Back" button or go back in your browser or use the breadcrumb
5. See your data in the form as initially entered
6. Modify values and resubmit
7. See modified values in the review page
8. Use "Submit" button to submit data for signup
9. You should see the thank you page and receive your validation email

## Help Requested

- Anything you think should be addressed as part of this
- Anything I missed

## Checklist
